mir_navigation/mir_2dexplore/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(mir_2dexplore)
+
+find_package(catkin REQUIRED)
+
+catkin_package(
+  CATKIN_DEPENDS
+    gmapping
+    cob_scan_unifier
```

Comment:
is this used somewhere?

Reply:
for slam. We want to have a option to unify the laser if we want. Right now it just uses front, but I think its better to have it than not.